automation/docker/run-tests.sh

```diff
@@ -75,8 +75,8 @@ docker run -e DOCKERHOST=$DOCKERHOST \
     --net=docker_default \
     -e ENV=$ENV \
     --add-host=firecloud-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} \
-    --add-host=firecloud-orchestration-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} \
-    --add-host=rawls.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} \
+    --add-host=firecloud-orchestration-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} \		
```

Comment:
Is this just some extra whitespace change? I think I remember seeing different changes here when I looked at the branch last week.